source/endpoints/parameters.rst

```diff
@@ -80,7 +80,81 @@ Examples:
    *  ``/users?sort=-username`` return users alphabetically from last to first by username (reverse order)
    *  ``/roles?sort=-id`` roles from last to first id
 
+.. _get-filters:
+
 Filters
 -------
 
-[TBD]
+Filters are one of the most powerful and versatile tools in BEdita API in order to retrieve only resources or objects you really need in your application.
+
+Filter expressions
+^^^^^^^^^^^^^^^^^^
+
+Generally every filter in a query string will have following syntax:
+
+  1. ``filter[{item}]={value}`` the simplest form, a filter item should be equal to some value
+  2. ``filter[{item}][{op}]={value}`` is a logical operation on filter value, where ``{op}`` can have following values:
+
+    * ``neq``, ``ne``, ``'!=`` or ``<>`` negation operator, meaning that filter item shoud not be equal to ``{{value}}``
+    * ``lt`` or ``<`` less than operator, meaning that filter item shoud be less than ``{{value}}``
+    * ``lte``, ``le`` or ``<=`` less than or equal operator; filter item shoud be less than or equal to ``{{value}}``
+    * ``gt`` or ``>`` greater than operator; filter item shoud be greater than ``{{value}}``
+    * ``gte``, ``ge`` or ``>=`` greater than or equal operator; filter item should be greater than or equal to ``{{value}}``
+
+  3. ``filter[{item}][]={value}`` array sintax: add a value to an array representing possible values of a filter item
+
+Filter expressions can be combined using ``&`` separator as many times as you want, limited only by the URL size.
+
+Too many ``filter`` combinations may of course result in unwantend or meaningless results, use them with caution :)
+
+Field filter
+^^^^^^^^^^^^
+
+The most common and simplest filter: retrieve only resources that have a field equal to some value, or greater/less than some value.
+
+Examples:
+
+   *  ``/users?filter[name]=Gustavo`` get users that have *Gustavo* as first ``name``
+   *  ``/objects?filter[id][gt]=100`` retrieve users with ``id`` greater than 100
```

Comment:
I'm not sure that "to retrieve" is the correct verb… I'd probably use "to return" instead.